violent-theremin/index.html

```diff
@@ -8,9 +8,10 @@
     <link href="styles/app.css" rel="stylesheet" type="text/css" />
   </head>
   <body>
-    <p class="start-message">
-      Click on the browser window or press the tab key to start the app.
-    </p>
+    <button class="start-message">
+      Click anywhere in the browser window <br />
+      to start the app.
```

Comment:
Not a big deal, but we can probably get rid of the `<br />` tag:

```suggestion
      Click anywhere in the browser window to start the app.
```

Reply:
Good catch. The `<br />` tag is not needed since the original font size is being used. It was added as a suggested breakpoint for the text when a different font size was set during development, but the original font size was ultimately committed in the PR and now spans the full screen on all sizes of simulated mobile browsers I tested. The `<br />` tag has been removed in https://github.com/mdn/webaudio-examples/pull/128/commits/8ab3dc3a8e683d111ac0ef926de382344b62732b.